编译时v2ray-plugin 出错

go: warning: "github.com/shadowsocks/v2ray-plugin/..." matched no packages

Building targets
no Go files in /home/magi/lede/build_dir/target-aarch64_generic_musl/v2ray-plugin-5.25.0/.go_work/build

make[3]: *** [Makefile:44: /home/magi/lede/build_dir/target-aarch64_generic_musl/v2ray-plugin-5.25.0/.built] Error 1
make[3]: Leaving directory '/home/magi/lede/feeds/small/v2ray-plugin'
time: package/feeds/small/v2ray-plugin/compile#0.14#1.16#4.84
    ERROR: package/feeds/small/v2ray-plugin failed to build.
make[2]: *** [package/Makefile:116: package/feeds/small/v2ray-plugin/compile] Error 1
make[2]: Leaving directory '/home/magi/lede'
make[1]: *** [package/Makefile:110: /home/magi/lede/staging_dir/target-aarch64_generic_musl/stamp/.package_compile] Error 2
make[1]: Leaving directory '/home/magi/lede'
make: *** [/home/magi/lede/include/toplevel.mk:231：world] 错误 2

go: warning: "github.com/shadowsocks/v2ray-plugin/..." matched no packages
2025-01-20T15:41:28.9990746Z 
2025-01-20T15:41:28.9990940Z Building targets
2025-01-20T15:41:29.0043443Z no Go files in /home/runner/work/lede/lede/build_dir/target-x86_64_musl/v2ray-plugin-5.25.0/.go_work/build
2025-01-20T15:41:29.0049028Z 
2025-01-20T15:41:29.0117410Z make[3]: *** [Makefile:44: /home/runner/work/lede/lede/build_dir/target-x86_64_musl/v2ray-plugin-5.25.0/.built] Error 1
2025-01-20T15:41:29.0118130Z make[3]: Leaving directory '/home/runner/work/lede/lede/feeds/kenzok8small/v2ray-plugin'
2025-01-20T15:41:29.0122856Z time: package/feeds/kenzok8small/v2ray-plugin/compile#0.07#0.12#0.17
2025-01-20T15:41:29.0130073Z     ERROR: package/feeds/kenzok8small/v2ray-plugin failed to build.
2025-01-20T15:41:29.0133868Z make[2]: *** [package/Makefile:116: package/feeds/kenzok8small/v2ray-plugin/compile] Error 1
2025-01-20T15:41:29.0138297Z make[2]: Leaving directory '/home/runner/work/lede/lede'
2025-01-20T15:41:29.0144801Z make[1]: *** [package/Makefile:110: /home/runner/work/lede/lede/staging_dir/target-x86_64_musl/stamp/.package_compile] Error 2
2025-01-20T15:41:29.0149381Z make[1]: Leaving directory '/home/runner/work/lede/lede'
2025-01-20T15:41:29.0156454Z make: *** [/home/runner/work/lede/lede/include/toplevel.mk:231: world] Error 2
2025-01-20T15:41:29.0183970Z ##[error]Process completed with exit code 2.

把golang替换成了1.23.5也是一样报这个错误

+1

Building targets
no Go files in /home/axx/lede/build_dir/target-x86_64_musl/v2ray-plugin-5.25.0/.go_work/build

make[3]: *** [Makefile:44: /home/axx/lede/build_dir/target-x86_64_musl/v2ray-plugin-5.25.0/.built] Error 1
make[3]: Leaving directory '/home/axx/lede/feeds/smpackage/v2ray-plugin'
time: package/feeds/smpackage/v2ray-plugin/compile#0.23#0.37#0.57
    ERROR: package/feeds/smpackage/v2ray-plugin failed to build.
make[2]: *** [package/Makefile:116: package/feeds/smpackage/v2ray-plugin/compile] Error 1
make[2]: Leaving directory '/home/axx/lede'
make[1]: *** [package/Makefile:110: /home/axx/lede/staging_dir/target-x86_64_musl/stamp/.package_compile] Error 2
make[1]: Leaving directory '/home/axx/lede'
make: *** [/home/axx/lede/include/toplevel.mk:231：world] 错误 2

--- a/v2ray-plugin/Makefile
+++ b/v2ray-plugin/Makefile
@@ -22,7 +22,7 @@ PKG_BUILD_PARALLEL:=1
 PKG_USE_MIPS16:=0
 PKG_BUILD_FLAGS:=no-mips16
 
-GO_PKG:=github.com/shadowsocks/v2ray-plugin
+GO_PKG:=github.com/teddysun/v2ray-plugin
 GO_PKG_LDFLAGS_X:=main.VERSION=v$(PKG_VERSION)
 
 include $(INCLUDE_DIR)/package.mk

